doc/api/child_process.md

```diff
@@ -290,6 +290,17 @@ parent process using the file descriptor (fd) identified using the
 environment variable `NODE_CHANNEL_FD` on the child process. The input and
 output on this fd is expected to be line delimited JSON objects.
 
+When specifying the [`stdio`][] option, the value *must* be a JSON Array
+containing at least one item with value `'ipc'` or a `TypeError` will be
```

Comment:
Shouldn't it be exactly one 'ipc'? I added test in ae4ce9fe73e9f65f00c6439f2c6ad7381d69d38e that fails if you try to setup multiple IPC channels.